qa/automation/CareerPortal.Tests.Scenarios/Send your CV/Send_your_CV.feature

```diff
@@ -0,0 +1,146 @@
+Feature: 1.1.2 Send_CV.feature
+
+Background: 
+	Given user accessed Quantori Career Portal from his browser
```

Comment:
What is the link for the Portal? What browser? You should provide specific data in Background